ipt_connect/IPTdev/tactics.py

```diff
@@ -0,0 +1,154 @@
+from django.shortcuts import render
+from django.views.decorators.cache import cache_page
+from models import *
+from django.contrib.auth.decorators import user_passes_test
+import parameters as params
+from views import *
+
+def build_tactics_for_two_teams(reporter_team, opponent_team):
```

Comment:
Not an issue, but a note: I think there might be some redundancy between this function and [Team.problem()](https://github.com/IPTnet/ipt_connect/blob/08e8bc4d0250b6050cadda5f08a63dd4e0c5c80f/ipt_connect/IPTdev/models.py#L429). You might want in the future to use only one piece of code for the two functions, so that if the rules change you only need to modify it in one place, not two.

Reply:
Opened #133